ota-plus-web/app/reactapp/src/containers/Preparation.jsx

```diff
@@ -52,25 +52,17 @@ class Preparation extends Component {
     constructor(props) {
         super(props);
         this.doorOpen = this.doorOpen.bind(this);
-        this.doubleCheckCreatedTuf = this.doubleCheckCreatedTuf.bind(this);
-        this.doubleCheckCreatedDirector = this.doubleCheckCreatedDirector.bind(this);
-        this.fetchDirectorRepoExists = this.fetchDirectorRepoExists.bind(this);
-        this.fetchTufRepoExists = this.fetchTufRepoExists.bind(this);
         this.userProfileHandler = new AsyncStatusCallbackHandler(props.userStore, 'userFetchAsync', this.checkUserProfile.bind(this));
         this.activatedProvisioningHandler = new AsyncStatusCallbackHandler(props.provisioningStore, 'provisioningStatusFetchAsync', this.checkActivatedProvisioning.bind(this));
-        this.createdTufHandler = new AsyncConflictCallbackHandler(props.packagesStore, 'tufRepoExistsFetchAsync', this.checkCreatedTuf.bind(this));
-        this.createTufHandler = new AsyncStatusCallbackHandler(props.packagesStore, 'tufRepoCreateFetchAsync', this.doubleCheckCreatedTuf.bind(this));
-        this.createdDirectorHandler = new AsyncConflictCallbackHandler(props.packagesStore, 'directorRepoExistsFetchAsync', this.checkCreatedDirector.bind(this));
-        this.createDirectorHandler = new AsyncStatusCallbackHandler(props.packagesStore, 'directorRepoCreateFetchAsync', this.doubleCheckCreatedDirector.bind(this));
+        this.createdTufHandler = new AsyncConflictCallbackHandler(props.provisioningStore, 'namespaceSetupFetchAsync', this.checkCreatedTuf.bind(this));

```

Comment:
The `setup` endpoint returns 200 with info in body, so don't need `AsyncConflictCallbackHandler`.